demos/m5/base/application_snapshot.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: ApplicationSnapshot
+metadata:
+  name: m5-snapshot
+  namespace: demo
+spec:
+  images:
+    - component: component1
+      pullSpec: pullspec1
```

Comment:
I know these are just essentially placeholders for now, but can we add some dummy digests to these? To prevent confusion from anyone who sees these